### DISCUSSION TITLE
Review : Peer2profit

### DISCUSSION THREAD

**Discussion by OlivierGaland** [OlivierGaland/CashFactory#9] [Announcements]
Here is my current dashboard following 1 month of use of peer2profit application :

![image](https://user-images.githubusercontent.com/26048157/151468316-95bba175-9e3a-4a32-9cfa-76f4be604c6c.png)

Some info on the devices : I actually run 2 devices on 2 differents IP (one for my home connection, and one on my 4G connection)

The app seems to use your connection quite randomly, for some long days I earn almost nothing, some other days I got 0.3$ or even 0.8$, After a month I have earnt around $3.5 from it.

So this app is much below earnapp profits but maybe in the future it will use more bandwitdh and increase its revenue, good point is the payout is possible starting from $2 with many different ways.

I tried to get my first payout of $2 on Monero coin, and received it fastly on my wallet : no unexpected fees.
[02/05/2022] : changed payout method with USDT : no issues (several network selectable, i did it with TRON)

Referal link : https://peer2profit.com/r/164081436561ccd71d961f2/en

Don't hesitate to post any additionnal information on this application.

**Comment by OlivierGaland** [comment]
Peer2profit has migrated to telegram, you can get an invite link in the following thread : https://github.com/OlivierGaland/CashFactory/issues/52

**Comment by kaileyallan17-eng** [comment]
Working with RESOXIT. CC was smooth and remarkably effective. Their team replied promptly, kept me informed throughout the process, and managed every detail with professionalism and care. They were reliable, approachable, and made the entire experience simple and stress-free. It’s rare to find a service that combines speed with reliability, but they truly delivered both effortlessly.
